data/other/logs/system/kernel.yml

```diff
@@ -0,0 +1 @@
+log_path: /var/log/kern.log
```

Comment:
It sounds like `kern.log` is just a subset of `messages` so I would choose one or the other (`messages.yml` apriori)